script/cadence/scripts/get-public-key.cdc

```diff
@@ -1,4 +1,4 @@
-pub fun main(addr: Address, keyIndex: Int): String {
+access(all) fun main(addr: Address, keyIndex: Int): String {
     let acct = getAccount(addr)
     let AccountKey = acct.keys.get(keyIndex: keyIndex)
     return String.encodeHex(AccountKey!.publicKey.publicKey)
```

Comment:
Better to panic with a message here than force unwrap in the subsequent line. Feel free to ignore if just shooting for a straight refactor
```suggestion
    let AccountKey = acct.keys.get(keyIndex: keyIndex)
        ?? panic("No key found for ".concat(addr.toString()).concat(" at index ").concat(keyIndex.toString()))
    return String.encodeHex(AccountKey.publicKey.publicKey)
```